### ISSUE TITLE
GUI client can't not show the rows of the table 

### ISSUE THREAD

**Issue by atifans** [pingcap/tidb#3991]
Please answer these questions before submitting your issue. Thanks!

1. What did you do?
The navicat client always shows rows and data length of 100, which cannot display these two values correctly. So are tools like heidisql.

2. What did you expect to see?
The GUI client displays the rows and data length values in the table correctly.

3. What did you see instead?
The GUI client displays the rows and data length values in the table correctly.

4. What version of TiDB are you using (`tidb-server -V`)?
Git Commit Hash: 6921161fde5c74c66d022f98ff9c793a0d37f148
UTC Build Time:  2017-08-01 03:35:43
navicat v12.0.x(MAC)

**Comment by shenli** [comment]
@atifans Thanks for your feedback! We are working on this. But it will take some time. We expect to fix this in this month.

**Comment by atifans** [comment]
Does it has been fixed?  
The navicat client still shows rows and data length of 100, which cannot display these two values correctly. 

./tidb-server -V
Release Version: 1.0.0
Git Commit Hash: 6e63f3890aec5c4d1a1a9bf1363db61d533cd24d
Git Commit Branch: master
UTC Build Time:  2017-10-17 11:11:19

**Comment by shenli** [comment]
@atifans This will be fixed in the next milestone 1.1.

**Comment by XuHuaiyu** [comment]
I'll take a look at it,
the version of your navicat is still `navicat v12.0.x(MAC)`?

**Comment by atifans** [comment]
yes,  the version of the navicat is still navicat v12.0.x(MAC)

**Comment by XuHuaiyu** [comment]
Hi, @atifans 
How can I re-produce this issue,
where will the navicat shows the rows and data length?
If possible, could you upload an image?

**Comment by mccxj** [comment]
@XuHuaiyu  I think I understand what the problem is. 
when you click table name, table info will show on the right side, `Rows` and `Auto Increment` always show 100.

**Comment by XuHuaiyu** [comment]
@mccxj Yes
Seems you're working on this?

**Comment by kolbe** [comment]
This issue seems to be resolved in TiDB 3.0.3. Confirmed in `show table status` output as well as in Navicat.

```
> show table status like 'trips'\G
*************************** 1. row ***************************
           Name: trips
         Engine: InnoDB
        Version: 10
     Row_format: Compact
           Rows: 19117643
 Avg_row_length: 118
    Data_length: 2271871215
Max_data_length: 0
   Index_length: 152941144
      Data_free: 0
 Auto_increment: 19220133
    Create_time: 2019-09-06 14:41:05
    Update_time: NULL
     Check_time: NULL
      Collation: utf8mb4_bin
       Checksum:
 Create_options:
        Comment:
1 row in set (0.004 sec)
```
